src/components/Books.js

```diff
@@ -0,0 +1,92 @@
+import { PropTypes } from 'prop-types';
+import { useDispatch } from 'react-redux';
+import { Progress } from 'antd';
+import { removeBook } from '../redux/books/books';
+
+const Books = ({
+  id, title, author, category,
+}) => {
+  const percent = Math.floor((Math.random() * 99) + 1);
+  const chapter = Math.floor((Math.random() * 25) + 1);
+  const dispatch = useDispatch();
+  const removeHandler = () => {
+    dispatch(removeBook(id));
+  };
+  return (
+    <li>
+      <div className="book">
+        <div className="content">
+          <div className="info">
+            <h4 className="category">{category}</h4>
+            <h2 className="title">{title}</h2>
+            <h6 className="author">{author}</h6>
+            <div className="action-buttons">
+              <button className="button-outline" type="button">
+                Comments
+              </button>
+              <div className="vertical-divider" />
+              <button
+                className="button-outline"
+                type="button"
+                onClick={removeHandler}
+              >
+                Remove
+              </button>
+              <div className="vertical-divider" />
+              <button className="button-outline" type="button">
+                Edit
+              </button>
+            </div>
+          </div>
+          <div className="progress-container">
+            <div className="progress">
+              <Progress
+                type="circle"
+                width={100}
+                format={() => ''}
+                strokeWidth={10}
+                strokeColor={{
+                  '0%': '#0291ff84',
+                  '100%': '#0290ff',
+                }}
+                trailColor="#e8e8e8"
+                percent={percent}
+              />
+            </div>
+            <div className="progress-stat">
+              <p className="percent-complete">
+                {percent}
+                %
+              </p>
+              <p className="completed">Completed</p>
+            </div>
+            <div className="progress-divider" />
+            <div className="current-chapter-container">
+              <div>
+                <p className="current-chapter-label">CURRENT CHAPTER</p>
+                <p className="current-chapter">
+                  Chapter
+                  {chapter}
+                </p>
+              </div>
+              <div>
+                <button className="btn-primary" type="button">
+                  UPDATE PROGRESS
+                </button>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+    </li>
```

Comment:
- [ ] [REQUIRED] You have done well here as well, but let's take a look at the container `book` class that houses the class `progress-container` and class `info`. The contents have a lot of padding/margin compared to what we have on Zeplin. Let's also stick with the font weight and other properties on zeplin. Click on each tag, and container on zeplin and you will have access to its styles on the right side of the screen, this will help guide your styles better. Let's fix these issues before requesting another review.

### Your Design
<img width="1333" alt="Screenshot 2023-02-11 at 17 59 07" src="https://user-images.githubusercontent.com/58692590/218270801-a4214a3b-60c4-482e-8ebe-ed2d92223e25.png">

### Zeplin Design
<img width="913" alt="Screenshot 2023-02-11 at 17 50 41" src="https://user-images.githubusercontent.com/58692590/218270818-748263a0-9df5-4385-9179-d098cac3246c.png">